src/journeys_utils.js

```diff
@@ -302,13 +302,9 @@ journeys_utils.addIframeOuterCSS = function(cssIframeContainer) {
 
 	if (cssIframeContainer) {}
 	else if (journeys_utils.position === 'top') {
-		var calculatedBodyMargin = +bannerMarginNumber + bodyMarginTopNumber;
-		document.body.style.marginTop = calculatedBodyMargin.toString() + 'px';
-	}
-	else if (journeys_utils.position === 'bottom') {
-		var calculatedBodyMargin = +bannerMarginNumber + bodyMarginBottomNumber;
-		document.body.style.marginBottom = calculatedBodyMargin.toString() + 'px';
-	}
+    var calculatedBodyMargin = +bannerMarginNumber + bodyMarginTopNumber;
```

Comment:
small spacing issue, sorry I don't think the linter works on this

Reply:
good catch, just fixed